wallet/wallet-node-client/src/node_traits.rs

```diff
@@ -57,4 +58,8 @@ pub trait NodeInterface {
     async fn p2p_get_connected_peers(&self) -> Result<Vec<ConnectedPeer>, Self::Error>;
     async fn p2p_add_reserved_node(&self, address: String) -> Result<(), Self::Error>;
     async fn p2p_remove_reserved_node(&self, address: String) -> Result<(), Self::Error>;
+
+    async fn get_all_mempool_transactions(
```

Comment:
This has to go, unfortunately. It encourages the bad practice of pulling all transactions in from the mempool. I see it isn't used anywhere.

Reply:
removed